packages/libretro/flycast/package.mk

```diff
@@ -18,31 +18,39 @@
 #  http://www.gnu.org/copyleft/gpl.html
 ################################################################################
 
-PKG_NAME="reicast"
-PKG_VERSION="2abb342"
+PKG_NAME="flycast"
+PKG_VERSION="4a020d4"
 PKG_REV="1"
-PKG_ARCH="arm i386 x86_64"
+PKG_ARCH="arm i386 x86_64 aarch64"
 PKG_LICENSE="GPLv2"
-PKG_SITE="https://github.com/libretro/reicast-emulator"
+PKG_SITE="https://github.com/libretro/flycast"
 PKG_GIT_URL="$PKG_SITE"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_PRIORITY="optional"
 PKG_SECTION="libretro"
-PKG_SHORTDESC="Reicast is a multiplatform Sega Dreamcast emulator"
-PKG_LONGDESC="Reicast is a multiplatform Sega Dreamcast emulator"
+PKG_SHORTDESC="Flycast is a multiplatform Sega Dreamcast emulator"
+PKG_LONGDESC="Flycast is a multiplatform Sega Dreamcast emulator"
 
 PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
 
 make_target() {
   if [ "$ARCH" == "arm" ]; then
-    make platform=rpi FORCE_GLES=1 HAVE_OPENMP=0
+    if [ "$OPENGLES_SUPPORT" = "yes" ]; then
+      REICAST_GLES=1
+    else
+      REICAST_GLES=0
+    fi
+
+    make platform=rpi FORCE_GLES=$REICAST_GLES HAVE_OPENMP=0
+  elif [ "$PROJECT" == "Switch" ]; then
+    make platform=arm64 HAVE_OPENMP=0 FORCE_GLES=0 GLES=0 HAVE_OIT=1
```

Comment:
I'm not sure about this tbh. I think I should add a generic one since our switch specific stuff is in another repo

Reply:
this would expose a rebase conflict here but it should be easy to resolve and add the switch-specific config when we get there.

Reply:
or should I leave off aarch64 altogether?

Reply:
No Switch stuff should be present here, take it out

Reply:
I suppose `"$PROJECT" == "aarch64"` is good enough for now, we don't support any aarch64 targets anyway, might as well remove it